extensions/cornerstone/src/commandsModule.ts

```diff
@@ -471,13 +471,8 @@ function commandsModule({
 
       const { viewport } = enabledElement;
 
-      if (viewport instanceof StackViewport) {
-        viewport.resetProperties();
-        viewport.resetCamera();
-      } else {
-        // Todo: add reset properties for volume viewport
```

Comment:
If I am not misaken, `resetProperties` was just added to the volume viewport, so I don't think you need the `if` here.